thrusterCalibration/runThrusters.ino

```diff
@@ -1,20 +1,23 @@
 #include <Servo.h>
-
+//Intialize Servos
 Servo servo2;
 Servo servo3;
 Servo servo4;
 Servo servo5;
 Servo servo6;
 Servo servo7;
-
+//Group Servos int pairs
 Servo thrusterGroup1[] = { servo2, servo3 };
 Servo thrusterGroup2[] = { servo4, servo5 };
 Servo thrusterGroup3[] = { servo6, servo7 };
-
+//array of trusters for all the ones on the robot
 Servo allThrusters[] = { servo2, servo3, servo4, servo5, servo6, servo7 };
-
+//stop value for thrushtes
 int thrusterStop = 1500;
-
+//commands
+/*
+Enums are just constants with predefined values if not defined it gose from 0 to n-1. n being the # of values
```

Comment:
I suppose this is fair...

BUT
1. 'goes' -> 'goes'
2. It's a hard habit that I have too, but don't include words like 'just' or 'simply' in comments.
3. There's no way we need a comment called 'Commands' for a declaration of something called `Commands`